src/system/UsersModule/Form/ConfigType/AuthenticationMethodsType.php

```diff
@@ -37,18 +40,21 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         ;
     }
 
+    /**
+     * {@inheritdoc}
+     */
     public function getBlockPrefix()
     {
         return 'zikulausersmodule_authenticationmenthodselector';
     }
 
     /**
-     * @param OptionsResolver $resolver
+     * {@inheritdoc}
```

Comment:
The braces define that it's inline. So you could mix it with an additional description. Otherwise it's the same I think.
https://www.phpdoc.org/docs/latest/guides/inheritance.html#the-inheritdoc-tag